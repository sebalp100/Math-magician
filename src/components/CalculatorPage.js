import React from 'react';
import Calculator from './Calculator';

const CalculatorPage = () => (
  <div className="calc-container">
    <h3>LetÂ´s do some math!</h3>
    <div className="box">
      <Calculator />
    </div>
  </div>
);

export default CalculatorPage;
