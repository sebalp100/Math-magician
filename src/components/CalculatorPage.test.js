import * as React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import CalculatorPage from './CalculatorPage';

test('First Snapshot test', () => {
  const component = renderer
    .create(<CalculatorPage />)
    .toJSON();

  expect(component).toMatchSnapshot();
});