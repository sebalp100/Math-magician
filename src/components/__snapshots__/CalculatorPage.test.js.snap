// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First Snapshot test 1`] = `
<div
  className="calc-container"
>
  <h3>
    Let´s do some math!
  </h3>
  <div
    className="box"
  >
    <div
      className="main-container"
    >
      <table
        cellSpacing="0"
      >
        <tbody>
          <tr>
            <td>
              <div
                className="output"
              >
                <div
                  className="input"
                >
                  0  
                </div>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              <button
                onClick={[Function]}
                type="submit"
              >
                AC
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                +/-
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                %
              </button>
              <button
                className="orange"
                onClick={[Function]}
                type="submit"
              >
                ÷
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <button
                onClick={[Function]}
                type="submit"
              >
                7
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                8
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                9
              </button>
              <button
                className="orange"
                onClick={[Function]}
                type="submit"
              >
                x
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <button
                onClick={[Function]}
                type="submit"
              >
                4
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                5
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                6
              </button>
              <button
                className="orange"
                onClick={[Function]}
                type="submit"
              >
                -
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <button
                onClick={[Function]}
                type="submit"
              >
                1
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                2
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                3
              </button>
              <button
                className="orange"
                onClick={[Function]}
                type="submit"
              >
                +
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <button
                className="zero"
                onClick={[Function]}
                type="submit"
              >
                0
              </button>
              <button
                onClick={[Function]}
                type="submit"
              >
                .
              </button>
              <button
                className="orange"
                onClick={[Function]}
                type="submit"
              >
                =
              </button>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
